cli/cli.rb

```diff
@@ -226,11 +233,13 @@ def start_services
     dns = Dnsmasq.new(machine)
     dns.up
     proxy = options[:proxy] || (options[:proxy].nil? && !proxy_disabled?)
-    if proxy
+    startup_containers = preferences[:startup_containers]
+    if proxy || startup_containers
       # this is hokey, but it can take a few seconds for docker daemon to be available
       # TODO: poll in a loop until the docker daemon responds
       sleep 5
-      HttpProxy.new(machine).up
+      HttpProxy.new(machine).up if proxy
```

Comment:
Could be as easy as passing in some options to the initialize of the class - volumes is already supported to pass into the container class.

Something like HttpProxy.new(machine,options[:proxy_volumes]).up